blocks/article-header/article-header.css

```diff
@@ -0,0 +1,19 @@
+.block .article-byline p {
+  display: inline;
+  margin-right: 20px;
+  font-size: 14px;
+  font-weight: 400;
+}
+
+
+.article-byline a:any-link {
```

Comment:
We could replace this selector here: https://github.com/hlxsites/servicenow/blob/main/styles/styles.css#L227 to not duplicate the link CSS.

We can keep the font family here and cleanup in `styles.css` https://github.com/hlxsites/servicenow/blob/main/styles/styles.css#L240-L242
```
font-family: var(--ff-gilroy-bold);
```

Reply:
I am not particularly fond of keeping block specific styles in global CSS, but I think this is not going to become too complex.